pkgs/servers/search/sphinxsearch/default.nix

```
@@ -31,6 +31,14 @@ stdenv.mkDerivation rec {
     expat
   ];
 
+  CXXFLAGS = stdenv.lib.optionals stdenv.isDarwin [
+    # see upstream bug: http://sphinxsearch.com/bugs/view.php?id=2578
+    # workaround for "error: invalid suffix on literal
+    "-Wno-reserved-user-defined-literal"
+    # workaround for "error: non-constant-expression cannot be narrowed from type 'long' to 'int'"
+    "-Wno-c++11-narrowing"
+  ];
```

Environment variables should be strings, see https://github.com/NixOS/nixpkgs/pull/72074

```suggestion
  CXXFLAGS = stdenv.lib.concatStringsSep " " (stdenv.lib.optionals stdenv.isDarwin [
    # see upstream bug: http://sphinxsearch.com/bugs/view.php?id=2578
    # workaround for "error: invalid suffix on literal
    "-Wno-reserved-user-defined-literal"
    # workaround for "error: non-constant-expression cannot be narrowed from type 'long' to 'int'"
    "-Wno-c++11-narrowing"
  ]);
```

---

whoops :)